src/Eshopworld.Messaging/Core/IMessenger.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
```

Comment:
I'm adding these to core library on https://github.com/eShopWorld/core/pull/31
Once merged I'll update